cranelift/umbrella/Cargo.toml

```diff
@@ -12,10 +12,30 @@ keywords = ["compile", "compiler", "jit"]
 edition.workspace = true
 
 [dependencies]
-cranelift-codegen = { workspace = true }
-cranelift-frontend = { workspace = true }
+cranelift-codegen = { workspace = true, optional = true }
```

Comment:
All other crates depend on cranelift-codegen, so I don't see any reason why you should be able to disable it.

Reply:
Makes sense! Updated!